go.mod

```diff
@@ -18,3 +27,78 @@ require (
 	sigs.k8s.io/gateway-api v0.4.0
 	sigs.k8s.io/yaml v1.3.0
 )
+
+require (
```

Comment:
What added all these indirect definitions? The switch to Go 1.16?

Reply:
Yes. Nit correction: switch to Go 1.17 (I know that's what you meant).